en-US/Audience.xml

```diff
@@ -17,6 +17,13 @@
 	</para>
 	<para>
         Other resources for technical writing are listed in <xref linkend="resources"/>.
+		Of these resources, <citetitle>IBM Style</citetitle> is now available for Red&nbsp;Hat employees to access online, but does not have a wider circulation.
```

Comment:
No need to anchor this statement in a specific point of time. "Now" will quickly become out of date! =D

Reply:
In cases where we mention "IBM Style" in general, what do you think about also providing a link to the homepage at https://www.ibm.com/docs/en/ibm-style? Currently, these references are styled as titles, but I'm wondering whether it would be more effective for our style guide (an online resource) to use hyperlinks for references that are also online resources. 

Rationale: Hyperlinks help users get the information that they need, at their point of need. Even for general mentions, including a link would be helpful. 

For example: 
Of these resources, [IBM Style](https://www.ibm.com/docs/en/ibm-style) 🔒 is available...